.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Yay for adding ESLint! Can I suggest you add it to your package.json by using `--save-dev` when installing? This way you can make sure people install the correct version local to the repo, and also makes it easy to set up in CI such as Travis, CircleCI of you so wish :smile: